README.markdown

```diff
@@ -322,8 +322,10 @@ load-nvmrc() {
   if [ -n "$nvmrc_path" ]; then
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
-    if [ "$nvmrc_node_version" != "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
+    if [ "$nvmrc_node_version" = "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
```

Comment:
the logic on this line says "if the nvmrc version is N/A and the node_version is not N/A", but `nvm version` doesn't print out `N/A` ever - so the second condition will always be true.